src/views/token-swap/components/swap-form/swap-form.view.jsx

```diff
@@ -1,13 +1,84 @@
 import React from 'react'
+import PropTypes from 'prop-types'
+import useSwapFormStyles from './swap-form.style'
+import { BigNumber } from 'ethers'
+import { ReactComponent as ArrowDown } from '../../../../images/icons/arrow-down-circle.svg'
+import BoxAmount from '../amount-box/amount-box.view'
+import { useLocation } from 'react-router-dom'
+function SwapForm ({
+  onGoToQuotes,
+  onOpenOfferSidenav,
+  accounts,
+  preferredCurrency,
+  fiatExchangeRates
+}) {
+  const classes = useSwapFormStyles()
 
-function SwapForm ({ onGoToQuotes, onOpenOfferSidenav }) {
+  const { search } = useLocation()
+  const urlSearchParams = new URLSearchParams(search)
+  const fromQuery = urlSearchParams.get('from')
+  const toQuery = urlSearchParams.get('to')
+  const [amount, setAmount] = React.useState(BigNumber.from(0))
+  const [isHide, setHide] = React.useState(false)
+  const [tokensSelected, setTokens] = React.useState({})
+  const setTokenPosition = tokenPosition => {
+    setTokens({ ...tokensSelected, ...tokenPosition })
+  }
+  React.useEffect(() => {
```

Comment:
it'd be nice to have some spacing betweent different hooks and functions to make the code more readable